### ISSUE TITLE
Add Files... Add Folder... Buttons are missing hover selections via Keyboard

### ISSUE THREAD

**Issue by chrisdaaz** [samvera/hyrax#3451]
### Descriptive summary

Hyrax Master -- 3.0.0 beta

### Rationale

This helps keyboard users add files and folders to works.

### Expected behavior

Users who tab throughout the edit work page should be able to notice when they have selected the `Add Files` and `Add Folder` buttons, like the `Add cloud files` button:

![image](https://user-images.githubusercontent.com/24395592/49243208-5f28ef00-f3d2-11e8-9a13-64a6f7414951.png)

### Actual behavior

Tabbing through that section of the page, it appears to the user that the `Add Files` and `Add Folder` buttons are skipped. They are not actually skipped; keyboard users can still add files and folders, but they don't see the same UI behavior for those buttons as they do for the `Add cloud files` button.

### Steps to reproduce the behavior

1. Visit the Add New or Edit Work page
2. Tab through the page to the Files section
3. Try to select the `Add files` or `Add folder` buttons

### Related work

Link to related tickets or prior related work here.

**Comment by mtribone** [comment]
Just posting the 2.0.0 issue for reference. https://github.com/samvera/hyrax/issues/1246

**Comment by kellyChex** [comment]
Confirmed this works on Nurax.

**Comment by dlim87** [comment]
QA - this is still working in nurax
